src/task.c

```diff
@@ -8,25 +8,32 @@
 
 #define MAX_HOOKS (2048)
 
-int              hook_init_done = 0;
-static int       num_hooks      = 0;
+int        hook_init_done = 0;
+static int num_hooks      = 0;
+// RESOLVE: At 8 bytes per task_hook{}, we are burning up 16K of memory. Only 2
+// hooks are currently being connected. What about reducing MAX_HOOKS to, say,
+// 8?
```

Comment:
This does feel a little much. Maybe something still hard to imagine blowing out like 64?